libs/dh/admin/feature-user-management/src/lib/drawer/dh-user-drawer.component.html

```diff
@@ -0,0 +1,34 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container *transloco="let t; read: 'admin.userManagement.drawer'">
+  <watt-drawer #drawer size="large" (closed)="onClose()">
+    <watt-drawer-topbar>
+      <watt-badge type="success">{{ t("active") }}</watt-badge>
+    </watt-drawer-topbar>
+
+    <watt-drawer-actions>
+      <watt-button variant="secondary">{{ t("edit") }}</watt-button>
+    </watt-drawer-actions>
+
+    <watt-drawer-content>
```

Comment:
suggestion: let's add `*ngIf="drawer.opened"` so the content is first initialized when the drawer is open.